src/ralph/lib/transitions/templates/transitions/run_transition.html

```diff
@@ -44,4 +49,15 @@
   <script type="text/javascript" src="{% static 'src/js/ralph-autocomplete.js' %}"></script>
   <script type="text/javascript" src="{% static 'vendor/js/foundation-datepicker.js' %}"></script>
   <script type="text/javascript" src="{% static 'src/js/foundation-datepicker-init.js' %}"></script>
+  <script type="text/javascript">
+    (function($) {
```

Comment:
please add some comment about it (ex. `automatically assign owner to be the same as user`)